console/src/modules/system/users/Binding.vue

```diff
@@ -51,7 +51,7 @@ function handleChangeType() {
 const isLoginType = computed(() => type.value !== "signup");
 </script>
 <template>
-  <div class="flex h-screen flex-col items-center bg-white/90 pt-[30vh]">
+  <div class="flex h-screen flex-col items-center bg-white/90 pt-[20vh]">
```

Comment:
```suggestion
  <div
    class="flex h-screen flex-col items-center bg-white/90 pt-[20vh] sm:pt-[30vh]"
  >
```

推荐通过这样的方式来适配分辨率，而不是直接修改这个值。